choreographer/src/main/java/eu/arrowhead/core/choreographer/database/service/ChoreographerSessionDBService.java

```diff
@@ -335,7 +401,7 @@ public ChoreographerSessionStep changeSessionStepStatus(final long sessionStepId
 				sessionStep = sessionStepRepository.saveAndFlush(sessionStep);
 				
 				final String exception = status == ChoreographerSessionStepStatus.ABORTED ? message : null;
-				worklog(sessionStep.getSession().getPlan().getName(), sessionStep.getStep().getAction().getName(), sessionStep.getStep().getName(), sessionStep.getSession().getId(),
+				worklog(sessionStep.getSession().getPlan().getName(), sessionStep.getStep().getAction().getName(), sessionStep.getStep().getName(), sessionStep.getSession().getId(), sessionStep.getSession().getExecutionNumber(),
```

Comment:
we should create a temp variable from sessionStep.getStep() and sessionStep.getSession() to made this line shorter